### ISSUE TITLE
Lint errors fail the build

### ISSUE THREAD

**Issue by oliverlockwood** [cyclestreets/android#167]
Since the upgrade to the latest Gradle plugin, there are some new lint errors that fail the build.

> ../../src/main/res/menu/itinerary_and_elevation_menu.xml:6: Should use android:showAsAction when not using the appcompat library
```
   3       xmlns:action="http://schemas.android.com/apk/res-auto">
   4   <item android:id="@+id/ic_menu_itinerary"
   5         android:title="@string/ic_menu_itinerary"
   6         action:showAsAction="ifRoom|withText"/>
   7   <item android:id="@+id/ic_menu_elevation"
   8         android:title="@string/ic_menu_elevation"
```

>../../src/main/res/menu/itinerary_and_elevation_menu.xml:9: Should use android:showAsAction when not using the appcompat library
```
   6         action:showAsAction="ifRoom|withText"/>
   7   <item android:id="@+id/ic_menu_elevation"
   8         android:title="@string/ic_menu_elevation"
   9         action:showAsAction="ifRoom|withText"/>
  10 </menu>
```

>../../src/main/res/menu/locations.xml:7: Should use android:showAsAction when not using the appcompat library
```
   4   <item android:id="@+id/ic_menu_addlocation"
   5         android:icon="@android:drawable/ic_menu_add"
   6         android:title="@string/ic_menu_addlocation"
   7         action:showAsAction="always|withText"/>
   8 </menu>
```

>../../src/main/res/menu/route_map.xml:7: Should use android:showAsAction when not using the appcompat library
```
   4   <item android:id="@+id/ic_menu_liveride"
   5         android:icon="@drawable/ic_menu_live_ride"
   6         android:title="@string/ic_menu_liveride"
   7         action:showAsAction="ifRoom|withText"/>
   8   <item android:id="@+id/ic_menu_directions"
   9         android:icon="@drawable/ic_menu_directions"
```

>../../src/main/res/menu/route_map.xml:11: Should use android:showAsAction when not using the appcompat library
```
   8   <item android:id="@+id/ic_menu_directions"
   9         android:icon="@drawable/ic_menu_directions"
  10         android:title="@string/ic_menu_directions"
  11         action:showAsAction="ifRoom|withText"/>
  12   <item android:id="@+id/ic_menu_saved_routes"
  13         android:icon="@drawable/ic_menu_places"
```

>../../src/main/res/menu/route_map.xml:15: Should use android:showAsAction when not using the appcompat library
```
  12   <item android:id="@+id/ic_menu_saved_routes"
  13         android:icon="@drawable/ic_menu_places"
  14         android:title="@string/ic_menu_saved_routes"
  15         action:showAsAction="never"/>
  16   <item android:id="@+id/ic_menu_route_number"
  17         android:icon="@drawable/ic_menu_route_number"
```

>../../src/main/res/menu/route_map.xml:19: Should use android:showAsAction when not using the appcompat library
```
  16   <item android:id="@+id/ic_menu_route_number"
  17         android:icon="@drawable/ic_menu_route_number"
  18         android:title="@string/ic_menu_route_number"
  19         action:showAsAction="never"/>
  20 </menu>
```

**Comment by oliverlockwood** [comment]
@jezhiggins I'm not familiar with the ins and outs of the appcompat library, so without taking time to dig (which I don't have immediately) I can't identify whether these are critical, can be fixed easily, should be suppressed, etc.

Can you look at this one to fix the builds?  Cheers 😃

**Comment by oliverlockwood** [comment]
Resolved by PR #168.
